assets/scripts/fe/src/app/components/public/login/login.component.ts

```diff
@@ -15,14 +15,16 @@ import { SlackService } from '../../../commons/services/auth/slack.service';
 })
 export class LoginComponent implements OnInit {
   private form : LoginForm;
-
+  private stateName = this.trans.params().next; // get statename from next= parameter
   constructor(
     private auth  : AuthService,
     private state : StateService,
-    private slack : SlackService
+    private slack : SlackService,
+    private trans : Transition
   ) { }
 
-  ngOnInit() {
+  ngOnInit() {   
+    console.log(this.trans);
```

Comment:
do not submit your changes to for review if it contains this kind of unneccessary codes